checker/mod_checking.ml

```diff
@@ -77,6 +77,20 @@ let check_constant_declaration env opac kn cb opacify =
   let opac = check_constant_declaration env opac kn cb opacify in
   Environ.add_constant kn cb env, opac
 
+let check_rewrite_rule env lab i (symb, rule) =
+  Flags.if_verbose Feedback.msg_notice (str "  checking rule:" ++ Label.print lab ++ str"#" ++ Pp.int i);
+  let { lhs_pat; rhs } = rule in
+  let symb_cb = Environ.lookup_constant symb env in
+  match symb_cb.const_body with Symbol unfix_fold ->
+    ignore unfix_fold
+  | _ -> ignore @@ invalid_arg "Rule defined on non-symbol";
+  ignore lhs_pat;
```

Comment:
Are these meant to be ignored? If so `_lhs_pat` is probably better.

Reply:
In my mind it's supposed to be a stub, making explicit the fact that the only reason the elements are ignored is because I don't know yet what will need to be checked

Reply:
I now do meaningful checks here